runtime/src/bank.rs

```diff
@@ -1146,7 +1151,7 @@ impl Bank {
         executed: &[Result<()>],
         tx_count: u64,
         signature_count: u64,
-    ) -> Vec<Result<()>> {
+    ) -> (Vec<Result<()>>, Vec<Result<()>>) {
```

Comment:
This is super hard to read.  wdyt about introducing a `struct` as a return type.